package:
    name: root
    version: 6.04.02
    #version: {{ environ.get('GIT_DESCRIBE_TAG', '') }} # bug, doesn't work for now: https://github.com/conda/conda-build/issues/357
source:
    #fn: root_v6.04.02.source.tar.gz
    #url: https://root.cern.ch/download/root_v6.04.02.source.tar.gz
    git_url: https://github.com/remenska/root
    git_tag: python3fixes
    #patches:
    #    - python3.patch
    #    - python3_set.patch
    #path: /home/daniela/anaconda_backup/anaconda/root-6.04.02
build:
    number: 13
    string: py{{ environ.get('PY_VER','') }}_gcc4.9.2
    detect_binary_files_with_prefix: True

requirements:
    build:
        - python  # [{{ environ.get('PY_VER') }}]     
        # - gcc 4.9.2
        #- libgcc >=4.8.4
        #- glibc 2.19*
        - openssl
        - qt
        - sqlite
        - fftw
        - pcre
        - libxml2
        - graphviz
        - cmake >=3.2.3
        - gsl 1.16
        - glibc 2.21
    run:
        #- glibc 2.19* ? really? maybe only for older linux/ubuntu distros
        - python  # [{{ environ.get('PY_VER') }}]
        - openssl
        - qt
        - sqlite
        - graphviz
        - libxml2
        - fftw
        - pcre
        # - libgcc-5 5.*
        - gsl 1.16
        - gcc 4.9.2
        - glibc 2.21

test:
    imports:

    commands:

about:
    home: http://root.cern.ch/
    license: LGPL v2.1

